### ISSUE TITLE
Can't open shortcuts from the context menu

### ISSUE THREAD

**Issue by dgrinin** [electrotype/vscode-windows-explorer-context-menu#1]
When I try to open a shortcut (.lnk file) using this extension, nothing happens.

Steps to represent:
1. Create a shortcut to some file. I tested with .html and .bat
2. Add folder with the shortcut to VSCode
3. Bring context menu on the shortcut (right click on the file --> Context Menu - Selected)
4. Click "Open"

The file that the shortcut points to should open, but it doesn't.
If I do the same in Windows Explorer, the file opens normally.

**Comment by electrotype** [comment]
Yes, some things don't work perfectly, sadly. Have you tried the 3 provided executables? But, even then, each of them had their own drawbacks when I tested them. The default one is the one working the best for me.

Maybe we should investigate if there have been updates on those executables (that I didn't develop myself), or if there are new alternatives. Feel free to investigate that and comment on this issue, if you want. If you find something interesting, I will have a look for sure.
